src/async-wrap.h

```diff
@@ -117,7 +117,7 @@ class AsyncWrap : public BaseObject {
 
   inline double get_trigger_id() const;
 
-  void AsyncReset(bool silent = false);
+  void AsyncReset(double eid = 0, bool silent = false);
```

Comment:
I think we should use `-1` as the value to check, this is also how the `triggerAsyncId` works in the public API. `0` as an `asyncId` has another meaning.